.circleci/config.yml

```diff
@@ -94,12 +94,12 @@ jobs:
       - restore_gems
       - run:
           name: Bundle install
-          command: bundle check --path $BUNDLE_PATH || bundle install --path $BUNDLE_PATH
+          command: bundle config set --local path 'vendor/bundle' && bundle install
```

Comment:
Added this due to the below warning:

```
[DEPRECATED] The `--path` flag is deprecated because it relies on being remembered across bundler invocations, which bundler will no longer do in future versions. Instead please use `bundle config set --local path 'vendor/bundle'`, and stop using this flag

```